proposals/NNNN-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -0,0 +1,143 @@
+# Adding new parameter of requiresAudioSupport and BluetoothDeviceAddress
+
+* Proposal: [SDL-NNNN](NNNN-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: Awaiting review
+* Impacted Platforms: [ Core / iOS / Java Suite / RPC / HMI ]
+
+## Introduction
+
+In this proposal, by adding the `requiresAudioSupport` in `RegisterAppInterface` and the `BluetooothDeviceAddress` in `DeviceInfo`.  
+when the device is connected via only USB, HU will connect to the SDL device BT automatically or to prompt the user for needing to BT connection.  
+
+## Motivation
+
+Since Android recommends not supporting AOA2.0, Android will no longer be able to play audio using only USB, and it will be necessary to use BT A2DP to play audio.  
+In the current Android library (java suite), if requiresAudioSupport is TRUE and BT A2DP is not connected,  
+SDL activation will be cancelled. Users can't notice why SDL App doesn't start, so UX needs to be improved.  
+To solve this problem, add a BluetoothDeviceAddress and requiresAudioSupport so that a new SDL device can be specified in RegisterAppInterface,  
```

Comment:
```suggestion
To solve this problem, add `BluetoothDeviceAddress` and `requiresAudioSupport` so that a new SDL device can be specified in RegisterAppInterface,
```